nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/SplitJson.java

```diff
@@ -61,6 +63,16 @@
         + "Each generated FlowFile is comprised of an element of the specified array and transferred to relationship 'split,' "
         + "with the original file transferred to the 'original' relationship. If the specified JsonPath is not found or "
         + "does not evaluate to an array element, the original file is routed to 'failure' and no files are generated.")
+@WritesAttributes({
+        @WritesAttribute(attribute = "fragment.identifier",
```

Comment:
Suggest a different name to indicate this is more of a fragment from a the same parent (e.g. fragment.parent.identifier or fragment.group.identifier)?  To me using fragment alone implies a unique id for that split instance (even though I know the attribute description explains more)